price_pusher/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pythnetwork/price-pusher",
-  "version": "5.4.6",
+  "version": "5.4.7",
```

Comment:
The version is bumped manually and the lock is not updated. 
Please update the versions using 
`npm version [x.y.z|patch|minor|major] --no-git-tag-add`